WordPress/src/main/java/org/wordpress/android/util/MediaUtilsWrapper.kt

```diff
@@ -58,4 +61,27 @@ class MediaUtilsWrapper @Inject constructor(private val appContext: Context) {
 
     fun isMimeTypeSupportedBySitePlan(site: SiteModel?, mimeType: String): Boolean =
             WPMediaUtils.isMimeTypeSupportedBySitePlan(site, mimeType)
+
+    @SuppressLint("InlinedApi")
+    fun isAllowedUploadVideoDuration(context: Context, uri: Uri): Boolean {
+        val mediaColumns = arrayOf(android.provider.MediaStore.Video.VideoColumns.DURATION)
+        val cursor: Cursor? = context.contentResolver.query(
+                uri,
+                mediaColumns,
+                null,
+                null,
+                null
+        )
+
+        cursor?.moveToFirst()
+        val durationColIndex: Int? = cursor?.getColumnIndexOrThrow(mediaColumns[0])
+        val fileDuration: Long = durationColIndex?.let { cursor.getLong(it) } ?: 0
+        cursor?.close()
```

Comment:
I wonder if we can use `MediaMetadataRetriever` here to get the duration (using [this key](https://developer.android.com/reference/android/media/MediaMetadataRetriever.html#METADATA_KEY_DURATION)). Wdyt?

Reply:
Updated to use MediaMetadataRetriever.  Thank you